src/System.Collections.NonGeneric/tests/Performance/System.Collections.NonGeneric.Performance.Tests.csproj

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <OutputType>Library</OutputType>
+    <RootNamespace>System.Collections.NonGeneric.Performance.Tests</RootNamespace>
+    <AssemblyName>System.Collections.NonGeneric.Performance.Tests</AssemblyName>
+    <ProjectGuid>{EE95AE39-845A-42D3-86D0-8065DBE56612}</ProjectGuid>
```

Comment:
This is the same GUID as the unit test project, probably a copy-paste issue.

While I'm commenting on this one, I'd recommend just removing the following properties from all of the new projects you created. I know they are probably also in the unit test projects and got copied from there, but you can just leave the existing projects alone and only clean them from the new projects.

* Configuration
* Platform
* OutputType
* ProjectGuid (VS might add this back in some cases, but just delete it to start out I think).